21-Hello-World/06-Type-Level-Programming/ReadMe.md

```diff
@@ -15,7 +15,7 @@ Let's explain a problem that highlights the third point: storing more informatio
 
 An `Array` is a very fast data structure, but it's problematic because we never know its exact size at compile-time. In other words, we don't get a compiler error if we reference an invalid spot in the array, and we won't know about the bug until it occurs when running the program. In short, this kind of function will always be a partial function.
 
-Here's an example in code. If the array is empty or only has `n - 1` elements, the function can only throw an error when we try to reference a non-existent element at index `n`. If the array has `n` or more elements, it can return that element.
+Here's an example in code. If the array is empty or only has fewer than `n - 1` elements, the function can only throw an error when we try to reference a non-existent element at index `n`. If the array has `n - 1` or more elements, it can return that element.
```

Comment:
This should probably be rendered as "...or has fewer than..." rather than the current "...or only has fewer than..."